m4_sessions/1803-inning/group_8.md

```diff
@@ -1,13 +1,36 @@
-Group 8
+## Group 8
 
-Group Members:
+### Group Members: 
+Spencer Herms, Lee Chow, Nyssa Keller, Ophus Wong
 
-Name of Session:
+### Name of Session: 
+Tech in Medicine: Balancing Ethics and Cutting Edge Technology
 
-Session Description (Why should students come to this session):
+### Session Description (Why should students come to this session): 
+- Technology is rapidly changing every aspect of our lives. As developers, we all understand the value of leveraging this to make our lives easier - medicine is no exception. Given the intimate involvement and ethical implications in the industry, it is important to explore this topic due to its widespread impact on many different walks of life. 
 
-Session Outcomes (What will students learn/leave with?):
+### Session Outcomes (What will students learn/leave with?):
+- Broader understanding of how technology affects the medical field
+- Help students develop and understand their own opinion regarding medtech
+- Give students an idea of what they would be working as a developer in medtech
 
-Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+### Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+  Intro/Housekeeping: 
+  - Group Members Intro
+  - Outline
+  - Decide: Big/small group discussion?
+  - Students: What interests you about tech in medicine?
+  
+  Topics: 
+  - Cutting Edge
+  - Ethical Implications
+  
+  Discussion: 
+  - Propose scenario: Discuss implications as a group
+  - Why does being ethical matter when you are trying to push the boundaries of medicine?
+  - What can you do as a developer to affect the outcome of technological advances in medicine?
+  
+  Closing/Summary:
+  - Group summary on white board to determine what everyone felt was the most important points in discussion for this session
```

Comment:
Excellent closing!  What next steps will they take?  What more do they want to do?